Makefile

```diff
@@ -23,16 +24,30 @@ ifeq ($(UNAME_S),Linux)
 	CGO_LDFLAGS := "$(CGO_LDFLAGS) -l:libjpeg.a"
 endif
 
-.PHONY: build-ffmpeg lint update-rdk module clean clean-all
+.PHONY: build-ffmpeg tool-install gofmt lint update-rdk module clean clean-all
 
 $(BIN_OUTPUT_PATH)/viamrtsp: build-ffmpeg *.go cmd/module/*.go
 	PKG_CONFIG_PATH=$(FFMPEG_BUILD)/lib/pkgconfig \
 		CGO_LDFLAGS=$(CGO_LDFLAGS) \
 		go build -o $(BIN_OUTPUT_PATH)/viamrtsp cmd/module/cmd.go
 
-lint:
+tool-install:
+	GOBIN=`pwd`/$(TOOL_BIN) go install \
+		github.com/edaniels/golinters/cmd/combined \
+		github.com/golangci/golangci-lint/cmd/golangci-lint \
+		github.com/rhysd/actionlint/cmd/actionlint
+
+gofmt:
 	gofmt -w -s .
 
+lint: gofmt tool-install
+	go mod tidy
+	export pkgs="`go list -f '{{.Dir}}' ./...`" && echo "$$pkgs" | xargs go vet -vettool=$(TOOL_BIN)/combined
+	GOGC=50 $(TOOL_BIN)/golangci-lint run -v --fix --config=./etc/.golangci.yaml
+
+unit-test:
```

Comment:
[nit] in RDK & other modules I've worked in at Viam we just have `make test` & it is clear that those are unit tests. I'd recommend we just call this `make test` & if we add support for integration tests in the future, call that `make integration-test`

Reply:
Done